frontend/src/components/object/ObjectUploadBasic.vue

```diff
@@ -51,12 +52,20 @@ const onSelectFile = () => {
 const onUpload = async () => {
   try {
     if( file.value ) {
+      toast.info('Upload starting...');
```

Comment:
i noticed there's a 'sticky' option for the toast popups. might be able to close it once the upload has finished.

Reply:
File upload starting...

I think a sticky toast option only makes sense if and only if the toast message contents were also dynamic in nature (aka seeing progress, indeterminate, etc). Otherwise having it just stay there is not likely to be a great user experience, especially if multiple files are being uploaded at once.